netlify.toml

```diff
@@ -131,1153 +115,1041 @@ to = "/account/my-organization/notifications/slack/slack-notifications"
-  status = 301
+from = "/publish-module/send-to-appstore"
+to = "/publish-integrations/ios-publish-integrations/send-to-app-store"
+status = 301
 
-# Swagger wrong (broken) URL on publish module API docs
 [[redirects]]
-  from = "/store"
-  to = "/publish-module"
-  status = 301
+from = "/publish-module/send-to-googleplay"
+to = "/publish-integrations/android-publish-integrations/publish-to-google-play"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/build-and-test/android-unit-tests"
-  to = "/workflows/android-specific-workflow-steps/android-unit-tests"
-  status = 301
+from = "/publish-integrations/android-publish-integrations/send-to-googleplay"
+to = "/publish-integrations/android-publish-integrations/publish-to-google-play"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/distribution/appdome-build-to-secure-for-android"
-  to = "/workflows/android-specific-workflow-steps/appdome-build-to-secure-for-android"
-  status = 301
+from = "/publish-module/send-to-huawei"
+to = "/publish-integrations/android-publish-integrations/publish-to-huawei-appgallery"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/distribution/bundle-universal-apk"
-  to = "/workflows/android-specific-workflow-steps/bundle-universal-apk"
-  status = 301
+from = "/publish-integrations/android-publish-integrations/send-to-huawei"
+to = "/publish-integrations/android-publish-integrations/publish-to-huawei-appgallery"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/build-and-test/gradle-runner"
-  to = "/workflows/android-specific-workflow-steps/gradle-runner"
-  status = 301
+from = "/publish-module/send-to-googleplay#deleting-android-publish-profiles"
+to = "/publish-module/creating-publish-profiles/managing-publish-profiles#delete-publish-profile"
+status = 301
 
 [[redirects]]
-  from = "/workflows/android-specific-workflow-steps/build-and-test/lint"
-  to = "/workflows/android-specific-workflow-steps/lint"
-  status = 301
+from = "/publish-integrations/android-publish-integrations/send-to-googleplay#deleting-android-publish-profiles"
+to = "/publish-module/creating-publish-profiles/managing-publish-profiles#delete-publish-profile"
+status = 301
 
 [[redirects]]
-  from = "/store-submit/google-play"
-  to = "/publish-module/send-to-googleplay"
-  status = 301
+from = "/publish-module/send-to-huawei#deleting-huawei-publish-profiles"
+to = "/publish-module/creating-publish-profiles/managing-publish-profiles#delete-publish-profile"
+status = 301
 
 [[redirects]]
-  from = "/store-submit/huawei-app-gallery"
-  to = "/publish-module/send-to-huawei"
-  status = 301
+from = "/publish-integrations/android-publish-integrations/send-to-huawei#deleting-huawei-publish-profiles"
+to = "/publish-module/creating-publish-profiles/managing-publish-profiles#delete-publish-profile"
```

Comment:
This redirect does not work, and this `from` is not ending with the `to` when tested on runtime. (https://github.com/appcircleio/appcircle-docusaurus/pull/717#discussion_r1817995559)